### ISSUE TITLE
lib.plot: Remove deprecated getters and setter functions (they've been replaced by properties)

### ISSUE THREAD

**Issue by dougthor42** [wxWidgets/Phoenix#519]
(This issue was created because PR #117 could not be finished in its entirety)

Many of the getters and setters in `lib.plot` have been deprecated and replaced with properties.

The old getters and setters can be removed when the deprecation period is complete (determed
by pkg maintainers).

Examples:
+ `SetXSpec`, `GetXSpec` --> `XSpec` property 
+ `SetFontSizeAxis`, `GetFontSizeAxis` --> `fontSizeAxis` property.

Removing these deprecated items will significantly reduce the LoC in `lib.plot.plotcanvas` and will make maintaining the `plot` package much easier.

**Comment by mesalu** [comment]
@RobinD42 When do you think would be a good release for removing the deprecated methods? 4.1?

**Comment by RobinD42** [comment]
Yes, probably sometime in the 4.1 series, but maybe not right away to give people a chance to make any needed changes in their code.
